### PR TITLE
"feat: added double tap to exit feature on Dashboard"

### PR DESCRIPTION
**Pull request by abhijitramesh**
Fixes #62

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by riyasingh1004** [comment]
Oh! But I saw the issue was still open.

**Comment by abhijitramesh** [comment]
Yes it is still pending to be reviewed 
Make sure that you get issues assigned before working on them 
please check out 
https://issues.apache.org/jira/browse/FINCN-13?jql=project%20%3D%20FINCN%20AND%20component%20%3D%20fineract-cn-mobile
and comment on the issues you are working on

File: app/src/main/java/org/apache/fineract/ui/online/DashboardActivity.java [hunk 1]
```diff
@@ -140,6 +144,19 @@ public void onBackPressed() {
         } else {
             super.onBackPressed();
         }
+        if (doubleBackToExitPressedOnce) {
+            super.onBackPressed();
+            return;
+        }
+        this.doubleBackToExitPressedOnce = true;
+        toast = Toast.makeText(this, "Please click BACK again to exit", Toast.LENGTH_SHORT);
```
Comment (miPlodder) [inline app/src/main/java/org/apache/fineract/ui/online/DashboardActivity.java:152]: No need to create a variable for Toast and don't hard-code the message.

File: app/src/main/java/org/apache/fineract/ui/online/DashboardActivity.java [hunk 2]
```diff
@@ -39,6 +41,8 @@
         NavigationView.OnNavigationItemSelectedListener {
 
     public static final String LOG_TAG = DashboardActivity.class.getSimpleName();
+    boolean doubleBackToExitPressedOnce = false;
+    private Toast toast = null;
```
Comment (miPlodder) [inline app/src/main/java/org/apache/fineract/ui/online/DashboardActivity.java:45]: Remove this

**Review by miPlodder — CHANGES_REQUESTED** [review]

**Comment by miPlodder** [comment]
@abhijitramesh Add Screenshots or GIF for UI changes.

**Comment by therajanmaurya** [comment]
Good Job, We have merged your changes here #64

**Pull request closed without merging**